main/main.c

```diff
@@ -636,6 +773,9 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.facility",		"LOG_USER",		PHP_INI_SYSTEM,		OnSetFacility,		syslog_facility,	php_core_globals,		core_globals)
+	STD_PHP_INI_ENTRY("syslog.ident",		"php",			PHP_INI_SYSTEM,		OnUpdateString,		syslog_ident,		php_core_globals,		core_globals)
+	STD_PHP_INI_BOOLEAN("have_called_openlog",	"0",			PHP_INI_SYSTEM,		NULL,			have_called_openlog,	php_core_globals,		core_globals)
```

Comment:
I don't want this to be settable, and indeed it should only be initialized once programmatically (i.e. in the binaries).  Are these the correct values?

Reply:
How does it get initialized without that?

Reply:
I would probably put it to the module startup - maybe somewhere around this line:

https://github.com/php/php-src/blob/3f53b02a53448ec5e7d2d0733668da92a9463730/main/main.c#L2137

But not sure if there is a better place - I don't think there is GINIT for PG but there might be a better place though :)

Reply:
Okay, changed as you suggested.